src/app/components/layout/navbar/navbar.component.html

```diff
@@ -244,7 +244,7 @@
                                 <i class="fa fa-sm fa-circle child-icon"></i>
                               </span>
                               <ng-container
-                                *ngIf="['northbound', 'southbound'].includes(s.type.toLowerCase()); else elseServiceTemplate">
+                                *ngIf="['northbound', 'southbound', 'dispatcher', 'notification', 'management', 'bucketstorage'].includes(s.type.toLowerCase()); else elseServiceTemplate">
```

Comment:
better to move `['northbound', 'southbound', 'dispatcher', 'notification', 'management', 'bucketstorage'].` array in ts. May be in utils.ts?

Reply:
This is being used at only one place (i.e. only here), so is this a better idea to move this array in utils.ts? What will be the benefit of it?